dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/configuration/JobParametersConfiguration.java

```diff
@@ -35,14 +35,15 @@ JobParameters jobParameters(Parameters parameters, DbsnpDatasource dbsnpDatasour
         parametersBuilder = new JobParametersBuilder();
 
         addParameter("assembly", parameters.getAssembly());
-        addParameter("assemblyTypes", String.join(",", parameters.getAssemblyTypes()));
         addParameter("batchId", parameters.getBatchId());
         addParameter("chunkSize", parameters.getChunkSize());
         addParameter("dbsnpBuild", parameters.getDbsnpBuild());
         addParameter("pageSize", parameters.getPageSize());
         addParameter("job", parameters.getJob());
         addParameter("variantsCollection", parameters.getVariantsCollection());
         addParameter("filesCollection", parameters.getFilesCollection());
+        addParameter("contigMappingUrl", parameters.getContigMappingUrl());
+        addParameter("referenceFastaFile", parameters.getReferenceFastaFile());
```

Comment:
Oh yes, if one run gets these wrong, it is absolutely mandatory that it restarts when new ones are selected!